src/main/starlark/builtins_bzl/bazel/java/bazel_java_binary.bzl

```diff
@@ -54,6 +54,8 @@ def _bazel_java_binary_impl(ctx):
     )
 
     if ctx.attr.use_testrunner:
+        if semantics.find_java_runtime_toolchain(ctx).version >= 17:
+            jvm_flags.append("-Djava.security.manager=allow");
```

Comment:
Could you please add a test for this (perhaps in com.google.devtools.build.lib.bazel.rules.java.JavaConfiguredTargetsTest)